packages/moonstone/styles/colors-light.less

```diff
@@ -224,4 +224,4 @@
 
 // Scrollable
 // ---------------------------------------
-@moon-scrollable-overscroll-color: @moon-accent;
+@moon-scrollable-overscroll-color: #454545;
```

Comment:
How about to assign variable rather than hardcoded value?

Reply:
The color is unique and it is similar to `@moon-dark-gray` but not exactly the same. So it is not a universal color but a specific color only for overscroll effect, in my opinion.